.config/example.yml

```diff
@@ -30,6 +30,9 @@ url: https://example.tld/
 # The port that your Misskey server should listen on.
 port: 3000
 
+# The hostname or address to listen on (default: all)
+#host: "127.0.0.1"
```

Comment:
The name "host" has a wide range and can be confusing, so how about "listenHost" or something similar?

Reply:
I think it is okay because I can not think of anything else it could mean in this context. It is simple and consistent with the node.js code.

If it's called "listenHost", then should "port" be called "listenPort"?

Reply:
Maybe `hostname` is better?

Reply:
When people see the property config.host, they don't know what host it refers to.

> then should "port" be called "listenPort"?

I think so.

Reply:
what about `bind` or `bindAddress`?

Reply:
This is a server configuration, so IMO host/port shouldn't be too confusing, especially with the comments here.

Reply:
(But then it could be confusing too because below `host` means the address the server connects to, and here it's to receive connections. Maybe bind is better then.)